### PR TITLE
Parse charges from SDF V2000 properties block

### PR DESCRIPTION
**Pull request by ajm13**

### CODE REVIEW

File: testing/unit/test_sdf.py [hunk 1]
```diff
@@ -84,6 +86,15 @@ def compare_atom_positions(complex1, complex2):
         assert(difference > -.001)
 
 
+def compare_atom_charges(complex1, complex2):
+    a1 = complex1.atoms
+    a2 = complex2.atoms
+    for a, _ in enumerate(complex1.atoms):
+        atom1 = next(a1)
+        atom2 = next(a2)
+        assert(atom1.formal_charge == atom2.formal_charge)
```
Comment (mjrosengrant) [inline testing/unit/test_sdf.py:95]:
```suggestion
def compare_atom_charges(complex1, complex2):
    for atom1, atom2 in zip(complex1.atoms, complex2.atoms):
        assert(atom1.formal_charge == atom2.formal_charge)
```

File: nanome/_internal/structure/io/sdf/parse.py [hunk 2]
```diff
@@ -80,8 +83,14 @@ def parse_model(lines):
                         bond.bond_order = record_chunk_int(line, 7, 9)
                         model.bonds.append(bond)
                         bond_counter = bond_counter - 1
-                    elif line[0] == 'm':
+                    elif line[0] == 'M':
```
Comment (mjrosengrant) [inline nanome/_internal/structure/io/sdf/parse.py:86]: would it make sense to make this `line[0].upper()`?

File: testing/unit/test_sdf.py [hunk 3]
```diff
@@ -99,6 +110,15 @@ def test_aromatic(self):
                 aromatic_bond = True
         assert(aromatic_bond)
 
+    def test_charges(self):
```
Comment (mjrosengrant) [inline testing/unit/test_sdf.py:113]: Can you add a docstring for this? With most of the work being done in read_write_read, its not obvious what this is testing

**Review by mjrosengrant — APPROVED** [review]
Looks like test_sdf.py could use a refactor, but the change itself is solid

File: nanome/_internal/structure/io/sdf/parse.py (hunk 2 above)
Comment (ajm13) [reply nanome/_internal/structure/io/sdf/parse.py:86]: nah, SDF standard is capital M

**Pull request merged**